recipes/evalml/meta.yaml

```diff
@@ -1,46 +1,87 @@
 {% set version = "0.12.2" %}
 
 package:
-  name: evalml
+  name: evalml-core
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/e/evalml/evalml-{{ version }}.tar.gz
   sha256: 1c4c440e9370f49ad21100908285a23e7c38fbc872feba34c0e6c47c39db8f76
 
-build:
-  entry_points:
-    - evalml = evalml.__main__:cli
-  noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+outputs:
+  - name: evalml-core
+    build:
+      entry_points:
+          - evalml = evalml.__main__:cli
+      noarch: python
+      number: 0
+      script: python -m pip install . --no-deps -vv
```

Comment:
You will need to put this command in a file and put the file name here. Also, always use ${PYTHON}

Reply:
I see - it was complaining about {{PYTHON}} not being defined before.

Reply:
yeah the script section on outputs is weird